nistats/.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
This one should be removed I guess

Reply:
The entire nistats directory will be removed in the 3rd PR which reorganizes the examples, so not going to worry about anything in that dir.